azurerm_app_insights/main.tf

```diff
@@ -13,3 +17,13 @@ resource "azurerm_application_insights" "azurerm_application_insights" {
   resource_group_name = "${data.azurerm_resource_group.rg.name}"
   application_type    = "${lookup(var.applications_insights[count.index], "azurerm_application_insights_application_type")}"
 }
+
+resource "azurerm_key_vault_secret" "azurerm_application_insights" {
+  name         = "${var.azurerm_application_insights_instrumentation_key}"
+  value        = "${azurerm_application_insights.azurerm_application_insights.instrumentation_key}"
+  key_vault_id = "${data.azurerm_key_vault.key_vault.id}"
+
+  lifecycle {
+    ignore_changes = ["value"]
+  }
```

Comment:
Why do we have this ignore_changes? We don't use it anywhere else when we create secrets..

Reply:
ignore_changes removed